### PR TITLE
rgriebl.brickstore version 2022.4.1.0

### PR DESCRIPTION
**Pull request by bebewold**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58220)

### CODE REVIEW

File: manifests/r/rgriebl/brickstore/2022.4.1.0/rgriebl.brickstore.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: rgriebl.brickstore
+PackageVersion: 2022.4.1.0
+PackageLocale: en-US
+Publisher: rgriebl
+PackageName: BrickStore
+License: opensource
```
Comment (Trenly) [inline manifests/r/rgriebl/brickstore/2022.4.1.0/rgriebl.brickstore.locale.en-US.yaml:9]:
```suggestion
License: GPL-2.0
LicenseUrl: https://github.com/rgriebl/brickstore/blob/main/LICENSE.GPL
```

File: manifests/r/rgriebl/brickstore/2022.4.1.0/rgriebl.brickstore.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: rgriebl.brickstore
+PackageVersion: 2022.4.1.0
+PackageLocale: en-US
+Publisher: rgriebl
+PackageName: BrickStore
+License: opensource
+ShortDescription: brickstore
```
Comment (Trenly) [inline manifests/r/rgriebl/brickstore/2022.4.1.0/rgriebl.brickstore.locale.en-US.yaml:10]:
```suggestion
ShortDescription: An offline BrickLink inventory management tool.
```

File: manifests/r/rgriebl/brickstore/2022.4.1.0/rgriebl.brickstore.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: rgriebl.brickstore
+PackageVersion: 2022.4.1.0
+PackageLocale: en-US
+Publisher: rgriebl
```
Comment (Trenly) [inline manifests/r/rgriebl/brickstore/2022.4.1.0/rgriebl.brickstore.locale.en-US.yaml:7]: This must match control panel
```suggestion
Publisher: https://brickforge.de/brickstore
```

File: manifests/r/rgriebl/brickstore/2022.4.1.0/rgriebl.brickstore.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: rgriebl.brickstore
+PackageVersion: 2022.4.1.0
```
Comment (Trenly) [inline manifests/r/rgriebl/brickstore/2022.4.1.0/rgriebl.brickstore.locale.en-US.yaml:5]: This must match control panel. When changing this value, don't forget to change the name of the version folder
```suggestion
PackageVersion: 2022.4.1
```

File: manifests/r/rgriebl/brickstore/2022.4.1.0/rgriebl.brickstore.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: rgriebl.brickstore
+PackageVersion: 2022.4.1.0
```
Comment (Trenly) [inline manifests/r/rgriebl/brickstore/2022.4.1.0/rgriebl.brickstore.installer.yaml:5]: This must match control panel. When changing this value, don't forget to change the name of the version folder
```suggestion
PackageVersion: 2022.4.1
```

File: manifests/r/rgriebl/brickstore/2022.4.1.0/rgriebl.brickstore.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: rgriebl.brickstore
+PackageVersion: 2022.4.1.0
```
Comment (Trenly) [inline manifests/r/rgriebl/brickstore/2022.4.1.0/rgriebl.brickstore.yaml:5]: This must match control panel. When changing this value, don't forget to change the name of the version folder
```suggestion
PackageVersion: 2022.4.1
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
> When changing this value, don't forget to change the name of the version folder

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/58220#issuecomment-1103475618

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**